R/get_nhdplushr.R

```diff
@@ -232,11 +232,12 @@ cull_cols <- function(x, keep_cols) {
 #'}
 make_standalone <- function(flowlines) {
```

Comment:
Changes here were required for the advanced network vignette which uses a different source of network data than I'd used before. Should all be backward compatible.